src/main/edit_delete/preProcess.ts

```diff
@@ -0,0 +1,49 @@
+import { Transcription } from 'sharedTypes';
+import JSONTranscription from './JSONTranscription';
+import TranscriptWord from './TranscriptWord';
+
+const preProcessTranscript = (
+  jsonTranscript: any,
+  duration: number
+): Transcription => {
+  const numberOfWords: number = jsonTranscript.words.length;
+
+  const processedTranscript: Transcription = new JSONTranscription(
+    [],
+    jsonTranscript.confidence
+  );
+
+  for (let i = 0; i < numberOfWords - 1; i += 1) {
```

Comment:
This one actually returns a word for each iteration so you can change it to a `map`